src/libraries/System.Net.Http/tests/FunctionalTests/MultipartContentTest.cs

```diff
@@ -476,28 +476,28 @@ public async Task ReadAsStreamAsync_CustomEncodingSelector_CustomEncodingIsUsed(
             }
 
             byte[] expected = Concat(
-                Encoding.Latin1.GetBytes("--fooBoundary\r\n"),
-                Encoding.Latin1.GetBytes("Content-Type: text/plain; charset=utf-8\r\n"),
-                Encoding.Latin1.GetBytes("latin1: "),
+                "--fooBoundary\r\n"u8.ToArray(),
+                "Content-Type: text/plain; charset=utf-8\r\n"u8.ToArray(),
+                "latin1: "u8.ToArray(),
                 Encoding.Latin1.GetBytes("\uD83D\uDE00"),
-                Encoding.Latin1.GetBytes("\r\n\r\n"),
-                Encoding.Latin1.GetBytes("bar1"),
-                Encoding.Latin1.GetBytes("\r\n--fooBoundary\r\n"),
-                Encoding.Latin1.GetBytes("utf8: "),
-                Encoding.UTF8.GetBytes("\uD83D\uDE00"),
-                Encoding.Latin1.GetBytes("\r\n\r\n"),
-                Encoding.Latin1.GetBytes("bar2"),
-                Encoding.Latin1.GetBytes("\r\n--fooBoundary\r\n"),
-                Encoding.Latin1.GetBytes("ascii: "),
+                "\r\n\r\n"u8.ToArray(),
+                "bar1"u8.ToArray(),
+                "\r\n--fooBoundary\r\n"u8.ToArray(),
+                "utf8: "u8.ToArray(),
+                "\uD83D\uDE00"u8.ToArray(),
```

Comment:
Nit: It reads strangely to me to see UTF-16 surrogate chars within a _u8_ block. I'd prefer to see us get in the habit of using `\U` syntax to write out the full scalar value and to recommend the same to our dev audience.

```suggestion
                "\U0001F600"u8.ToArray(),
```

Reply:
Ok